### PR TITLE
Clarify Max logic

### PR DESCRIPTION
**Pull request by Joeysantoro**
When opening a pull request to submit a new EIP, please use the suggested template: https://github.com/ethereum/EIPs/blob/master/eip-template.md

We have a GitHub bot that automatically merges some PRs. It will merge yours immediately if certain criteria are met:

 - The PR edits only existing draft PRs.
 - The build passes.
 - Your GitHub username or email address is listed in the 'author' header of all affected PRs, inside <triangular brackets>.
 - If matching on email address, the email address is the one publicly listed on your GitHub profile.

### CODE REVIEW

**Review by fubuloubu — APPROVED** [review]

File: EIPS/eip-4626.md [hunk 1]
```diff
@@ -91,11 +91,10 @@ MUST _NOT_ revert.
 
 #### maxDeposit
 
-Total number of underlying assets that `caller` can deposit.
+Total number of underlying assets that `to` can deposit, where `to` corresponds to the input parameter of a `deposit` call.
```
Comment (alcueca) [inline EIPS/eip-4626.md:94]:
```suggestion
Total number of underlying assets that can be deposited in `to` vault balance, where `to` corresponds to the input parameter of a `deposit` call.
```

File: EIPS/eip-4626.md [hunk 2]
```diff
@@ -170,11 +169,10 @@ Note that most implementations will require pre-approval of the Vault with the V
 
 #### maxMint
 
-Total number of underlying shares that `caller` can mint.
+Total number of underlying shares that `to` can mint, where `to` corresponds to the input parameter of a `mint` call.
```
Comment (alcueca) [inline EIPS/eip-4626.md:172]:
```suggestion
Total number of underlying shares that can be minted for `to`, where `to` corresponds to the input parameter of a `mint` call.
```

File: EIPS/eip-4626.md [hunk 3]
```diff
@@ -249,11 +247,9 @@ Note that most implementations will require pre-approval of the Vault with the V
 
 #### maxWithdraw
 
-Total number of underlying assets that `caller` can withdraw.
+Total number of underlying assets that `from` can withdraw, where `from` corresponds to the input parameter of a `withdraw` call.
```
Comment (alcueca) [inline EIPS/eip-4626.md:250]:
```suggestion
Total number of underlying assets that can be withdrawn from `from` vault balance, where `from` corresponds to the input parameter of a `withdraw` call.
```

File: EIPS/eip-4626.md [hunk 4]
```diff
@@ -331,11 +327,10 @@ Those methods should be performed separately.
 
 #### maxRedeem
 
-Total number of underlying shares that `caller` can redeem.
+Total number of underlying shares that `from` can redeem, where `from` corresponds to the input parameter of a `redeem` call.
```
Comment (alcueca) [inline EIPS/eip-4626.md:330]:
```suggestion
Total number of underlying shares that can redeemed from `from` vault balance, where `from` corresponds to the input parameter of a `redeem` call.
```

**Review by alcueca — CHANGES_REQUESTED** [review]
The language needs to be changed since `to` and `from` can't be expected to be the caller.

**Pull request merged**